ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -51,14 +51,15 @@ struct S3FileSystem;
 // Statistics to be collected for the S3 storage.
 struct S3Statistics {
     // Operations using AWS SDK.
-    uint64_t listObjectsCount;  // Number of S3 list objects requests
-    uint64_t putObjectCount;    // Number of S3 put object requests
-    uint64_t getObjectCount;    // Number of S3 get object requests
-    uint64_t objectExistsCount; // Number of S3 object exists requests
+    std::atomic<int64_t> listObjectsCount;         // Number of S3 list objects requests
+    std::atomic<int64_t> putObjectCount;           // Number of S3 put object requests
+    std::atomic<int64_t> objectExistsAndSizeCount; // Number of S3 object exists requests
```

Comment:
Maybe we can just call it objectExistsCount, I think I renamed it as there was a separate objectExists call different from getting the size, which no longer exists and is reaplaced with read call.

Reply:
changed [fa8365c](https://github.com/wiredtiger/wiredtiger/pull/9708/commits/fa8365ce9ebfbe01a182b327672b7475457c9f95)